pallets/parachain-staking/src/lib.rs

```diff
@@ -168,6 +170,8 @@ pub mod pallet {
 		/// Handler to notify the runtime when a new round begin.
 		/// If you don't need it, you can specify the type `()`.
 		type OnNewRound: OnNewRound;
+		/// Whether a given collator has completed required registration to be selected as block author
+		type CollatorRegistration: ValidatorRegistration<Self::AccountId>;
```

Comment:
you gotta pass this in inside of the NEU/TUR runtimes tho, right? just reminder not to forget to put that into the OAK blockchain repo when you update the new version! (and double check me here too, but i think we're pulling off an existing tag for our moonbeam repo rather than just main/master branch)

Reply:
yup, thanks for the reminder. Going to cut that pr now.